app/views/gov_one/info.html.slim

```diff
@@ -1,14 +1,27 @@
-= render 'learning/cms_debug'
-
 - content_for :page_title do
-  = html_title t('gov_one_info.title')
+  = html_title t('home.title')
+
+- content_for :hero do
+  .govuk-grid-row class='govuk-!-padding-top-9 govuk-!-padding-bottom-0'
+    .govuk-grid-column-three-quarters
+      h1.dfe-heading-xl class='govuk-!-margin-bottom-4'
+        = t('gov_one_info.hero_header')
+      p.govuk-body-l = t('gov_one_info.hero_body')
+
 
 .govuk-grid-row
-  .govuk-grid-column-one-half
-    = m('gov_one_info.body')
+  .govuk-grid-column-three-quarters
+    . class='govuk-!-margin-bottom-5'
+        = m('gov_one_info.body')
+        hr
+
+    = login_button
 
-    - if current_user
-      = logout_button
-    - else
-      = login_button
+. class='govuk-!-margin-top-6 govuk-!-margin-bottom-9'
```

Comment:
This needs to be inside a govuk row with a control on the width:
<div class="govuk-grid-row govuk-!-margin-top-4 govuk-!-margin-bottom-9">
    <div class="govuk-grid-column-three-quarters"><details class="govuk-details" data-module="govuk-details"><summary class="govuk-details__summary"><span class="govuk-details__summary-text">How to access an existing training account</span></summary><div class="govuk-details__text">If you have an existing early years child development training account but you do not yet have a GOV.UK One Login, you must use the same email address for both accounts. This will ensure that any progress you have made through the training is retained.</div></details></div></div>

Reply:
Cheers Rick, my latest changes are in this commit [3b7ba6a](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/968/commits/3b7ba6a7d3cbae5a3158c4a5ed1553e4f7086c90)